ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json

```diff
@@ -286,6 +286,13 @@
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Add access to the lable

Reply:
Removed access to the ones given as the role doesn't have access to any of the other placement event fields